src/__tests__/builder.ts

```diff
@@ -1,21 +1,21 @@
 export interface IBuilder<T> {
-  with<K extends keyof T>(key: K, value: T[K]): IBuilder<T>;
+  with<K extends keyof T>(key: K, value: T[K]): this;
```

Comment:
Due to the scope of the `IBuilder` and how it's used in many tests I would create a PR just with these changes.

Reply:
As per discussion, I've created a [separate PR](https://github.com/safe-global/safe-client-gateway/pull/947) to tackle this.